dashboard/lti_new.py

```diff
@@ -57,7 +57,7 @@ def lti_error(error_message: Any) -> JsonResponse:
 class LTIException(Exception):
```

Comment:
LTIException is used only for generate_jwks() and get_jwks().

Since #1223 , MyLA config uses the generic canvas instance urls, and jwks-related instructions have been removed from MyLA wiki, I wonder whether we should remove the above two function and hence the LTIException definition after all.

Reply:
Right, these are only used when using the LTI services (like grading service) which we aren't currently using. So these could be just cleaned up.

I was also wondering if we could use [LTIServiceException](https://github.com/dmitry-viskov/pylti1.3/blob/7474685aecc35d528b944caf05b28ea66ac6d204/pylti1p3/exception.py#L16) from the library instead of our own exception here. But I wasn't sure the type of the **config** variable. 

This seems fine now, maybe create another issue to remove those jwks methods.